### PR TITLE
updating service->cloud

### PR DESCRIPTION
**Pull request by susanev**
this does not update the pulumi service provider, as that name has not yet changed

see also 
https://github.com/pulumi/pulumi-hugo/pull/2666
https://github.com/pulumi/examples/pull/1412
https://github.com/pulumi/pulumi/pull/12630

### CODE REVIEW

File: themes/default/content/registry/packages/pulumiservice/installation-configuration.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The Pulumi Service provider is available as a package in all Pulumi languages:
 
 ## Setup
 
-To provision resources with the Pulumi Service provider, you need to have Pulumi Service credentials. Pulumi Service maintains documentation on how to create access tokens [here](https://www.pulumi.com/docs/intro/pulumi-service/accounts/#access-tokens).
+To provision resources with the Pulumi Service provider, you need to have Pulumi Service credentials. Pulumi Service maintains documentation on how to create access tokens [here](https://www.pulumi.com/docs/intro/pulumi-cloud/accounts/#access-tokens).
```
Comment (susanev) [inline themes/default/content/registry/packages/pulumiservice/installation-configuration.md:18]:
```suggestion
To provision resources with the Pulumi Service provider, you must have Pulumi Cloud credentials. Pulumi Cloud maintains documentation on creating [access tokens](https://www.pulumi.com/docs/intro/pulumi-cloud/accounts/#access-tokens).
```

File: themes/default/content/registry/packages/pulumiservice/installation-configuration.md [hunk 2]
```diff
@@ -62,5 +62,5 @@ Use `pulumi config set pulumiservice:<option>` or pass options to the [construct
 
 | Option | Required/Optional | Description |
 |-----|------|----|
-| `accessToken`| Required | [Pulumi Service Access Tokens](https://www.pulumi.com/docs/intro/pulumi-service/accounts/#access-tokens) |
+| `accessToken`| Required | [Pulumi Service Access Tokens](https://www.pulumi.com/docs/intro/pulumi-cloud/accounts/#access-tokens) |
```
Comment (susanev) [inline themes/default/content/registry/packages/pulumiservice/installation-configuration.md:65]:
```suggestion
| `accessToken`| Required | [Pulumi Cloud Access Tokens](https://www.pulumi.com/docs/intro/pulumi-cloud/accounts/#access-tokens) |
```

**Comment by scottslowe** [comment]
General question about these changes: should we be using "the Pulumi Cloud" instead of just "Pulumi Cloud"? If we are going to capitalize (as a proper noun/product name), the latter seems like the correct usage to me.

File: themes/default/content/registry/packages/pulumiservice/installation-configuration.md [hunk 3]
```diff
@@ -62,5 +62,5 @@ Use `pulumi config set pulumiservice:<option>` or pass options to the [construct
 
 | Option | Required/Optional | Description |
 |-----|------|----|
-| `accessToken`| Required | [Pulumi Service Access Tokens](https://www.pulumi.com/docs/intro/pulumi-service/accounts/#access-tokens) |
+| `accessToken`| Required | [Pulumi Cloud Access Tokens](https://www.pulumi.com/docs/intro/pulumi-cloud/accounts/#access-tokens) |
 | `apiUrl`| Optional | Allows overriding default [Pulumi Service API URL](https://www.pulumi.com/docs/reference/service-rest-api) for [self hosted customers](https://www.pulumi.com/docs/guides/self-hosted/).
```
Comment (scottslowe) [inline themes/default/content/registry/packages/pulumiservice/installation-configuration.md:66]:
```suggestion
| `apiUrl`| Optional | Allows overriding default [Pulumi Cloud API URL](https://www.pulumi.com/docs/reference/cloud-rest-api) for [self hosted customers](https://www.pulumi.com/docs/guides/self-hosted/).
```

File: themes/default/content/registry/packages/aws/how-to-guides/ec2-webserver.md [hunk 4]
```diff
@@ -525,7 +525,7 @@ Do you want to perform this update? details
 
 Select `yes` to confirm the update.
 
-You can use `pulumi stack output` to get the value of stack outputs from the CLI.  To do so, `curl` the EC2 instance to confirm that the HTTP server is running. Stack outputs can also be viewed in the Pulumi Service.
+You can use `pulumi stack output` to get the value of stack outputs from the CLI.  To do so, `curl` the EC2 instance to confirm that the HTTP server is running. Stack outputs can also be viewed in the Pulumi Cloud.
```
Comment (jkodroff) [inline themes/default/content/registry/packages/aws/how-to-guides/ec2-webserver.md:528]:
```suggestion
You can use `pulumi stack output` to get the value of stack outputs from the CLI.  To do so, `curl` the EC2 instance to confirm that the HTTP server is running. Stack outputs can also be viewed in the Pulumi Cloud console.
```

**Review by jkodroff — CHANGES_REQUESTED** [review]
Same idea here as the previous PR: Need to check for usages of "the Pulumi Cloud" and reword depending on context.

**Comment by susanev** [comment]
@scottslowe 

>General question about these changes: should we be using "the Pulumi Cloud" instead of just "Pulumi Cloud"? If we are going to capitalize (as a proper noun/product name), the latter seems like the correct usage to me.

no standard has been established yet, so i wont be taking on standardizing it in this work. but if you'd like to take that on, pls do!

File: themes/default/content/registry/packages/aws/how-to-guides/ec2-webserver.md (hunk 4 above)
Comment (susanev) [reply themes/default/content/registry/packages/aws/how-to-guides/ec2-webserver.md:528]: @jkodroff this does not append console in the original so lets leave this as is.

File: themes/default/content/registry/packages/aws/how-to-guides/ec2-webserver.md [hunk 5]
```diff
@@ -282,9 +282,9 @@ Permalink: https://app.pulumi.com/bermudezmt/myproject/webserver-dev/updates/1
 
 ### Step 4: View your stack resources
 
-#### **Pulumi Service**
+#### **Pulumi Cloud**
 
-To see the full details of the deployment and the resources that are now part of the stack, open the update link in a browser. The **Resources** tab in the Pulumi Service has a link to the AWS console for the provisioned EC2 instance.
+To see the full details of the deployment and the resources that are now part of the stack, open the update link in a browser. The **Resources** tab in the Pulumi Cloud has a link to the AWS console for the provisioned EC2 instance.
```
[the remark this answers is not in the document]
Comment (susanev) [reply themes/default/content/registry/packages/aws/how-to-guides/ec2-webserver.md:287]: @jkodroff this does not append console in the original, so lets leave this as is.

File: scripts/ci/sync.sh [hunk 6]
```diff
@@ -67,7 +67,7 @@ aws s3 cp "$build_dir/latest-version" "${destination_bucket_uri}/latest-version"
 #
 # Why use a local file and not `pulumi config`, or some other persistence store? Because
 # we need ensure that every CI job deploys only what it was responsible for building.
-# Coupled with the locking we get from the Pulumi Service, using a local file is a safe
+# Coupled with the locking we get from the Pulumi Cloud, using a local file is a safe
```
[the remark this answers is not in the document]
Comment (susanev) [reply scripts/ci/sync.sh:70]: @jkodroff this does not append console in the original, so lets leave this as is.

**Review by jkodroff — COMMENTED** [review]
Removing request changes

**Comment by scottslowe** [comment]
@susanev 

> no standard has been established yet, so i wont be taking on standardizing it in this work

I guess I'm a bit confused as to why we would make this change (from "Service" to "Cloud") and *not* address the correct/desired usage of our product name at the same time? Seems like more work to have to come back and do this all over again (particularly when you have two reviewers providing the same feedback), but I'll defer to you.

**Review by anita-trimbur — APPROVED** [review]

File: themes/default/content/registry/packages/aws/how-to-guides/rest-api.md [hunk 7]
```diff
@@ -104,7 +104,7 @@ For a more interesting view that shows the result of calling a Lambda function,
 
 ### Step 5: Manage the stack
 
-The output also contained a permalink to the Pulumi Service. Select that link to review the stack in the web UI, examine logs and resource usage, and learn how you can invite friends and coworkers to collaborate on stacks.
+The output also contained a permalink to the Pulumi Cloud. Select that link to review the stack in the web UI, examine logs and resource usage, and learn how you can invite friends and coworkers to collaborate on stacks.
```
Comment (jkodroff) [inline themes/default/content/registry/packages/aws/how-to-guides/rest-api.md:107]:
```suggestion
The output also contained a permalink to the Pulumi Cloud console. Select that link to review the stack in the web UI, examine logs and resource usage, and learn how you can invite friends and coworkers to collaborate on stacks.
```

File: themes/default/content/registry/packages/aws/how-to-guides/s3-website.md [hunk 8]
```diff
@@ -111,7 +111,7 @@ Current stack outputs (1):
     bucketName                                     s3-website-bucket-8533d8b
 ```
 
-To see that the S3 bucket objects exist, you can either use the AWS Console (which is accessible from the Pulumi Service) or the AWS CLI:
+To see that the S3 bucket objects exist, you can either use the AWS Console (which is accessible from the Pulumi Cloud) or the AWS CLI:
```
Comment (jkodroff) [inline themes/default/content/registry/packages/aws/how-to-guides/s3-website.md:114]:
```suggestion
To see that the S3 bucket objects exist, you can either use the AWS Console (which is accessible from the Pulumi Cloud console) or the AWS CLI:
```

File: themes/default/content/registry/packages/azure/how-to-guides/container-webserver.md [hunk 9]
```diff
@@ -138,7 +138,7 @@ Before moving on, let's tear down the resources that are part of our stack.
 
 1. Run `pulumi destroy` to tear down all resources.  You'll be prompted to make sure you really want to delete these resources. This takes some time; Pulumi waits for the ACI to shutdown and for the resource group to be removed before it considers the destroy operation to be complete.
 
-1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the Pulumi Service.
+1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the Pulumi Cloud.
```
Comment (jkodroff) [inline themes/default/content/registry/packages/azure/how-to-guides/container-webserver.md:141]:
```suggestion
1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the Pulumi Cloud console.
```

File: themes/default/content/registry/packages/gcp/how-to-guides/gce-webserver.md [hunk 10]
```diff
@@ -189,7 +189,7 @@ Pulumi program to define the new state we want our infrastructure to be in, then
     Permalink: https://app.pulumi.com/lukehoban/webservergcp-dev/updates/2
     ```
 
-1. We can use `pulumi stack output` to get the value of stack outputs from the CLI.  So we can `curl` the virtual machine instance to see the HTTP server running there. Stack outputs can also be viewed on the Pulumi Service.
+1. We can use `pulumi stack output` to get the value of stack outputs from the CLI.  So we can `curl` the virtual machine instance to see the HTTP server running there. Stack outputs can also be viewed on the Pulumi Cloud.
```
Comment (jkodroff) [inline themes/default/content/registry/packages/gcp/how-to-guides/gce-webserver.md:192]:
```suggestion
1. We can use `pulumi stack output` to get the value of stack outputs from the CLI.  So we can `curl` the virtual machine instance to see the HTTP server running there. Stack outputs can also be viewed ine the Pulumi Cloud console.
```

File: themes/default/content/registry/packages/gcp/how-to-guides/gce-webserver.md [hunk 11]
```diff
@@ -202,7 +202,7 @@ Before moving on, let's tear down the resources that are part of our stack.
 
 1. Run `pulumi destroy` to tear down all resources.  You'll be prompted to make sure you really want to delete these resources. This takes about 60 seconds; Pulumi waits for the virtual machine instance to shutdown and for the compute network to be removed before it considers the destroy operation to be complete.
 
-1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the Pulumi Service.
+1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the Pulumi Cloud.
```
Comment (jkodroff) [inline themes/default/content/registry/packages/gcp/how-to-guides/gce-webserver.md:205]:
```suggestion
1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the Pulumi Cloud console.
```

File: themes/default/content/registry/packages/kubernetes/faq.md [hunk 12]
```diff
@@ -125,7 +125,7 @@ const wordpress = new k8s.helm.v3.Chart("wpdev", {
 
 See the [pulumi/examples][examples] repo for more details.
 
-## Does the Pulumi Service see my credentials in the kubeconfig file?
+## Does the Pulumi Cloud see my credentials in the kubeconfig file?
```
Comment (jkodroff) [inline themes/default/content/registry/packages/kubernetes/faq.md:128]:
```suggestion
## Does Pulumi Cloud see my credentials in the kubeconfig file?
```

File: themes/default/content/registry/packages/kubernetes/faq.md [hunk 13]
```diff
@@ -136,12 +136,12 @@ same resources concurrently, and stores the state history of your deployments.
 If you ask Pulumi to boot up a cluster (_e.g._, as our [EKS example][eks] shows), then it is
 possible to retrieve a kubeconfig file _for that cluster only_. In this case, the kubeconfig file is
 stored as part of the state history of rollouts -- but they are considered opaque state by the
```
Comment (jkodroff) [inline themes/default/content/registry/packages/kubernetes/faq.md:138]:
```suggestion
stored as part of the state history of rollouts -- but they are considered opaque state by 
```

File: themes/default/content/registry/packages/kubernetes/faq.md [hunk 14]
```diff
@@ -136,12 +136,12 @@ same resources concurrently, and stores the state history of your deployments.
...
-Pulumi service. The service **never** uses these values for any reason whatsoever.
+Pulumi Cloud. The service **never** uses these values for any reason whatsoever.
```
Comment (jkodroff) [inline themes/default/content/registry/packages/kubernetes/faq.md:139]:
```suggestion
Pulumi Cloud. Pulumi Cloud **never** uses these values for any reason whatsoever.
```

File: themes/default/content/registry/packages/kubernetes/faq.md [hunk 15]
```diff
@@ -136,12 +136,12 @@ same resources concurrently, and stores the state history of your deployments.
...
 
 Additionally, the contents of the kubeconfig files are not sensitive for managed clusters
 on the major cloud providers, as they defer authentication to their local CLIs.
 
-## Does the Pulumi Service see my cloud provider credentials?
+## Does the Pulumi Cloud see my cloud provider credentials?
```
Comment (jkodroff) [inline themes/default/content/registry/packages/kubernetes/faq.md:144]:
```suggestion
## Does Pulumi Cloud see my cloud provider credentials?
```

File: themes/default/content/registry/packages/kubernetes/how-to-guides/aks.md [hunk 16]
```diff
@@ -380,7 +380,7 @@ Before moving on, tear down the resources that are part of your stack.
 
 1. Run `pulumi destroy` to tear down all resources.  You'll be prompted to make sure you really want to delete these resources. This takes some time; Pulumi waits for all the resources to be removed before it considers the destroy operation to be complete.
 
-1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the Pulumi Service.
+1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the Pulumi Cloud.
```
Comment (jkodroff) [inline themes/default/content/registry/packages/kubernetes/how-to-guides/aks.md:383]:
```suggestion
1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the Pulumi Cloud console.
```

File: themes/default/content/registry/packages/kubernetes/how-to-guides/eks-migrate-nodegroups.md [hunk 17]
```diff
@@ -441,7 +441,7 @@ $ pulumi destroy
 $ pulumi stack rm
 ```
 
-> **Note:** This command deletes all deployment history from the Pulumi Service and cannot be undone.
+> **Note:** This command deletes all deployment history from the Pulumi Cloud and cannot be undone.
```
Comment (jkodroff) [inline themes/default/content/registry/packages/kubernetes/how-to-guides/eks-migrate-nodegroups.md:444]:
```suggestion
> **Note:** This command deletes all deployment history from Pulumi Cloud and cannot be undone.
```

File: themes/default/content/registry/packages/kubernetes/how-to-guides/eks.md [hunk 18]
```diff
@@ -393,7 +393,7 @@ Run the following command to tear down the resources that are part of our stack.
 
 1. Run `pulumi destroy` to tear down all resources.  You'll be prompted to make sure you really want to delete these resources.
 
-1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the Pulumi Service and cannot be undone.
+1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the Pulumi Cloud and cannot be undone.
```
Comment (jkodroff) [inline themes/default/content/registry/packages/kubernetes/how-to-guides/eks.md:396]:
```suggestion
1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from Pulumi Cloud and cannot be undone.
```

File: themes/default/content/registry/packages/kubernetes/how-to-guides/gke.md [hunk 19]
```diff
@@ -381,7 +381,7 @@ Run the following command to tear down the resources that are part of our stack.
     resources.
 
 1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from the
```
Comment (jkodroff) [inline themes/default/content/registry/packages/kubernetes/how-to-guides/gke.md:383]:
```suggestion
1. To delete the stack itself, run `pulumi stack rm`. Note that this command deletes all deployment history from
```

File: themes/default/content/registry/packages/kubernetes/installation-configuration.md [hunk 20]
```diff
@@ -62,7 +62,7 @@ Verify the cluster is configured and up by running `kubectl get pods`.
 
 Then follow the [getting started guide](/docs/get-started/kubernetes) or explore some [code examples](https://github.com/pulumi/examples#kubernetes).
 
-Note: Pulumi **never** sends **any** authentication [secrets](/docs/intro/concepts/secrets/) or credentials to the Pulumi service.
+Note: Pulumi **never** sends **any** authentication [secrets](/docs/intro/concepts/secrets/) or credentials to the Pulumi Cloud.
```
Comment (jkodroff) [inline themes/default/content/registry/packages/kubernetes/installation-configuration.md:65]:
```suggestion
Note: Pulumi **never** sends **any** authentication [secrets](/docs/intro/concepts/secrets/) or credentials to Pulumi Cloud.
```

File: themes/default/content/registry/packages/kubernetes/installation-configuration.md [hunk 21]
```diff
@@ -118,7 +118,7 @@ to the constructor of a `new kubernetes.Provider` to construct a specific instan
 
 ### kubectl proxy
 
-Each Kubernetes resource managed by Pulumi will have a link in the [Pulumi Service](https://app.pulumi.com")
+Each Kubernetes resource managed by Pulumi will have a link in the [Pulumi Cloud](https://app.pulumi.com")
```
Comment (jkodroff) [inline themes/default/content/registry/packages/kubernetes/installation-configuration.md:121]:
```suggestion
Each Kubernetes resource managed by Pulumi will have a link in the [Pulumi Cloud console](https://app.pulumi.com")
```

File: themes/default/content/registry/packages/pulumiservice/installation-configuration.md [hunk 22]
```diff
@@ -15,7 +15,7 @@ The Pulumi Service provider is available as a package in all Pulumi languages:
 
 ## Setup
 
-To provision resources with the Pulumi Service provider, you need to have Pulumi Service credentials. Pulumi Service maintains documentation on how to create access tokens [here](https://www.pulumi.com/docs/intro/pulumi-service/accounts/#access-tokens).
+To provision resources with the Pulumi Service provider, you must have Pulumi Cloud credentials. Pulumi Cloud maintains documentation on creating [access tokens](https://www.pulumi.com/docs/intro/pulumi-cloud/accounts/#access-tokens).
```
Comment (jkodroff) [inline themes/default/content/registry/packages/pulumiservice/installation-configuration.md:18]: Should this be a relref?

File: themes/default/content/registry/packages/pulumiservice/installation-configuration.md [hunk 23]
```diff
@@ -62,5 +62,5 @@ Use `pulumi config set pulumiservice:<option>` or pass options to the [construct
 
 | Option | Required/Optional | Description |
 |-----|------|----|
-| `accessToken`| Required | [Pulumi Service Access Tokens](https://www.pulumi.com/docs/intro/pulumi-service/accounts/#access-tokens) |
-| `apiUrl`| Optional | Allows overriding default [Pulumi Service API URL](https://www.pulumi.com/docs/reference/service-rest-api) for [self hosted customers](https://www.pulumi.com/docs/guides/self-hosted/).
+| `accessToken`| Required | [Pulumi Cloud Access Tokens](https://www.pulumi.com/docs/intro/pulumi-cloud/accounts/#access-tokens) |
```
Comment (jkodroff) [inline themes/default/content/registry/packages/pulumiservice/installation-configuration.md:65]: Should this be a relref?

**Review by jkodroff — COMMENTED** [review]
Added suggested changes. Will commit in a batch right after.

File: themes/default/content/registry/packages/pulumiservice/installation-configuration.md (hunk 23 above)
Comment (susanev) [reply themes/default/content/registry/packages/pulumiservice/installation-configuration.md:65]: @jkodroff no, nothing should be a relref, we removed support for relrefs. but you can drop the pulumi.coms if you'd like

File: themes/default/content/registry/packages/pulumiservice/installation-configuration.md (hunk 22 above)
Comment (susanev) [reply themes/default/content/registry/packages/pulumiservice/installation-configuration.md:18]: @jkodroff no, nothing should be a relref, we removed support for relrefs. but you can drop the pulumi.coms if you'd like

**Review by sean1588 — APPROVED** [review]

**Pull request merged**